Makefile-docker

```diff
@@ -148,13 +148,12 @@ setup-ui-tests:
 	# Create a proper superuser that can be used to access the API
 	python manage.py waffle_switch super-create-accounts on --create
 	python manage.py waffle_switch activate-autograph-signing on --create
-	python manage.py createsuperuser --email=uitest@mozilla.com --username=uitest --noinput --add-to-supercreate-group --save-api-credentials=tests/ui/variables.json --hostname=olympia-frontend.test
+	python manage.py createsuperuser --email=uitest@restmail.net --username=uitest --noinput --add-to-supercreate-group --save-api-credentials=tests/ui/variables.json --hostname=olympia-frontend.test
```

Comment:
If we're not making use of the saved variables anymore we should remove that functionality imho.

Reply:
Remove it from the api?

Reply:
Also, yes.

Reply:
This seems still todo?

This needs to be removed from the management command, there is nothing in the API actually. So `src/olympia/users/management/commands/createsuperuser.py` is the only place where the variable saving should be removed from.